tidy3d/components/monitor.py

```diff
@@ -32,6 +32,23 @@ class Monitor(Box, ABC):
         min_length=1,
     )
 
+    interval_space: Tuple[Literal[1], Literal[1], Literal[1]] = pydantic.Field(
```

Comment:
One question: why using "Literal" here rather than pd.PostiveInt?

Reply:
That's what it is for monitors in which it can actually be set, it is overwritten [here](https://github.com/flexcompute/tidy3d/blob/3e757803bd9c13bfed3f7eb85df27f88460aaa73/tidy3d/components/monitor.py#L237-L246). `Literal` means that it cannot be anything else than 1. This is how it is for most monitors. It just helps unify things and not have to do things like `if isinstance(...)` when dealing with downsampling.